### PR TITLE
Add CogView::cog_view_[set|is]_visible() accessors

### PR DESCRIPTION
**Pull request by psaavedra**
Add convenient accessors to the "visible" property the to the **CogView** external API:

* `CogView::cog_view_set_visible()` sets the view as the one visible inside of the viewport where the view is attached.
* `CogView::cog_view_is_visible()` gets if the view is visible inside of the viewport where the view is attached.

### CODE REVIEW

File: core/cog-view.c [hunk 1]
```diff
@@ -454,3 +454,51 @@ cog_view_get_viewport(CogView *self)
...
+
+/**
+ * cog_view_set_visible:
+ * @self: A view.
+ *
+ * Set the view as the one visible inside of the viewport where the view is
+ * attached.
+ *
+ * Returns: True if the view succesfully set as visible in its viewport and
+ * false in any other case.
```
Comment (aperezdc) [inline core/cog-view.c:489]: Maybe the following is easier to parse (well, it is for me):

```suggestion
 * Set the view as visible in its viewport.
 *
 * Returns: %TRUE if the view was set as visible, or %FALSE if the view
 *    does is not attached to a viewport.
```

File: core/cog-view.c [hunk 2]
```diff
@@ -454,3 +454,51 @@ cog_view_get_viewport(CogView *self)
...
+ *
+ * Since: 0.20
+ */
+gboolean
+cog_view_set_visible(CogView *self)
+{
+    g_assert(COG_IS_VIEW(self));
+
+    g_autoptr(CogViewport) viewport = cog_view_get_viewport(self);
+    g_return_val_if_fail(viewport != NULL, false);
```
Comment (aperezdc) [inline core/cog-view.c:499]: I would return `FALSE` here if there is no viewport. If we want to consider that it is a programmer error to call this function when the view is not attached to a viewport, then the function does not need to return any value and use `g_return_if_fail()` here.

But I think it's better to do:

```suggestion
    if (!viewport)
        return FALSE;
```

File: core/cog-view.c [hunk 3]
```diff
@@ -454,3 +454,51 @@ cog_view_get_viewport(CogView *self)
...
+
+    cog_viewport_set_visible_view(viewport, self);
+
+    return true;
```
Comment (aperezdc) [inline core/cog-view.c:503]: The `true` value is for the `bool` type, but for `gboolean` we must use `TRUE`:

 ```suggestion
    return TRUE;
```

**Review by aperezdc — CHANGES_REQUESTED** [review]
I agree on the idea for the new API, but I have a few change requests. If you look into them, I think we can merge this today. Thanks @psaavedra! 🙏🏼

**Comment by psaavedra** [comment]
> I agree on the idea for the new API, but I have a few change requests. If you look into them, I think we can merge this today. Thanks @psaavedra! 🙏🏼

Done.

**Review by aperezdc — APPROVED** [review]

**Pull request merged**